docs/faq/end-users.md

```diff
@@ -1,49 +1,84 @@
 
-We expect integration with other wallets and independent implementations to appear.
+Withdrawals take place first on the Hermez L2, where the fee is paid in the same token as the transaction.  Next, it needs to move back to L1 which requires "gas" to cover the L1 transaction fee (this amount is determined by current prices on Ethereum Network. Hermez has no control over this fee).  If your L1 Ethereum account does not have enough funds to cover the gas charge, then the withdrawal cannot be completed and will produce the error message.  
 
-## Can coordinators take my money?
+To remedy this, please make sure you have enough funds in your L1 Ethereum account to cover the gas to complete the withdrawal.
 
-No.
+# Why is my Deposit taking so long to process?
 
-## Can coordinators censor transactions?
+Deposits move funds from Ethereum  L1 to L2 (Hermez) and must be forged by the Coordinator.  Depending on where the Coordinator is in the forging process, deposits may be as fast as 45 seconds or could take up to 15 minutes maximum. 
 
-Coordinators can decide which transactions they select to include in the batch to forge.
+# Can I use my Ledger/Trezor hardware with my Hermez Wallet?
 
-In case that a coordinator censures a transaction, users have the option to force coordinators to include their transaction. This is done by sending an on-chain transaction (`forceExit` or `forceTransfer`)
+Currently hardware wallets are not supported on Hermez, but should be supported in the near future. 
+
+# Fees and Tokens
+
+# Who makes money from the transaction fees?
+
+Transaction fees are collected by the Coordinator who placed the highest bid in the Coordinator auction.
+
+# How much will transactions cost on Hermez?
+
+Fees are used to pay for the infrastructure required to maintain the Hermez Coordinator plus the L1 costs involved in forging a new batch of transactions. Cost of the transaction will be split among the hundreds (or even thousands) of transactions in the batch, resulting in a lower cost compared to L1.
+
+# How will a new Hermez account cost?
+
+Creating a new Hermez account involves an L1 transaction and an L2 transaction. L1 transactions
+need to pay for gas, and L2 transaction need to pay the Coordinator fees.
+
+# Which tokens can I use on Hermez Network?
+
+For the current list of tokens registered in Hermez Network please visit: https://explorer.hermez.io/tokens
+
+# Why is the gas fee so high? I thought Hermez was supposed to lower the cost of transactions.
+
+The "gas" fee to move funds from L2 back to Ethereum mainnet (L1) is determined by current prices on the Ethereum Network. Hermez has no control over this fee.
+
+# Do I need HEZ tokens to use the Hermez Network?
+
+There's no need for HEZ tokens on the user side to send transactions.  HEZ is only required for coordinators to bid in the auction.
```

Comment:
There's no need for HEZ tokens on the user side to send transactions as fees are paid in the same token of the transaction.